website/source/docs/guides/backup.html.md

```diff
@@ -0,0 +1,87 @@
+ 
+First, we will run the basic snapshot command with the all the defaults, including `consistent` mode.
+
+```sh
+$ consul snapshot save backup.snap
+Saved and verified snapshot to index 1176
+```
+The backup will be saved locally in the directory where we ran the command. 
+
+You can view metadata about the backup with the `inspect` subcommand. 
+
+```sh
+$ consul snapshot inspect backup.snap
+ID           2-1182-1542056499724
+Size         4115
+Index        1182
+Term         2
+Version      1
+```
+
+To understand each field review the inspect [documentation](https://www.consul.io/docs/commands/snapshot/inspect.html). Notably, the `Version` field does not correspond to the version of the data. Rather it is the snapshot format version. 
+
+Next, let’s collect the cluster data from a non-leader by specifying stale mode.
+
+```sh
+$ consul snapshot save -stale backup.snap
+Saved and verified snapshot to index 2276
+```
+
+Once ACLs and agent certificates are configured, they can be passed in with the following flags.
+
+```sh
+$ consul snapshot save -stale -token=<ead6787-23thbd-666789> -ca-file=</path/to/file> backup.snap
+Saved and verified snapshot to index 2287
+```
+
+Alternatively, you can use environment variables to say the ACL token and CA cert. 
+
+For production use, the  `snapshot save` command or [API](https://www.consul.io/api/snapshot.html) should be scripted and run frequently. In addition to frequently backing up the cluster state, there are several use cases when you would also want to manually execute `snapshot save`. First, it should be best practice to backup the cluster before upgrading. If the upgrade does not go according to plan, this will ensure you have the latest data. Second, if the cluster loses quorum it may be beneficial to save the state before the non-leaders become divergent. Finally, you can manually snapshot the cluster for use of bootstrapping a new cluster. 
+
+Operationally, the backup process does not need to be executed on every server. Additionally, you can use the configuration options to save the backups to a mounted filesystem. The mounted filesystem can even be cloud storage, such as Amazon S3. The enterprise command `snapshot agent` automates this process.
+
+## Restore from Backup 
+
+Running the `restore` process should be straightforward. However, there are a couple actions you can take to ensure the process goes smoothly. First, make sure the cluster you are restoring is stable. Second, you will only need to run the process once, on the leader. The cluster gossip protocol will ensure the data is propagated to the non-leaders.  
+
+The basic restore command. 
+
+```sh
+$ consul snapshot restore backup.snap
+Restored snapshot
+```
```

Comment:
Should we state _where_ this command is running? It needs to be run on a server otherwise the CONSUL_HTTP_ADDR or -http-addr options need to be specified.

Reply:
We do, but I will re-org to make it clearer.